api_tests/base/test_serializers.py

```diff
@@ -97,6 +101,53 @@ def test_expected_serializers_have_meta_types(self):
                 ser.Meta, 'get_type'
             ), 'Serializer {} has no Meta.type_ or Meta.get_type()'.format(ser)
 
+    def test_serializers_types_are_kebab_case(self):
+        serializers = JSONAPISerializer.__subclasses__()
+        request = make_drf_request_with_version(version=KEBAB_CASE_VERSION)
+        for serializer in serializers:
+            if serializer == WaffleSerializer or serializer == BaseWaffleSerializer:
+                continue
+            if serializer == SchemaSerializer:
+                for schema_serializer in serializer.__subclasses__():
+                    if 'Deprecated' not in str(schema_serializer):
+                        if hasattr(serializer.Meta, 'get_type'):
+                            json_type = serializer.Meta.get_type(request)
+                        else:
+                            json_type = serializer.Meta.type_
+                        assert '_' not in json_type
+                        assert json_type == json_type.lower()
+            if not re.match('^(api_test|test).*', serializer.__module__):
+                if hasattr(serializer.Meta, 'get_type'):
+                    json_type = serializer.Meta.get_type(request)
+                else:
+                    json_type = serializer.Meta.type_
+                assert '_' not in json_type
+                assert json_type == json_type.lower()
+
+    def test_deprecation_warning_for_2_15_snake_case(self):
```

Comment:
While you're here, can you just remove the version number out of this test, as it's not accurate - but will likely get out of date again!